### PR TITLE
Window size defaults to fullscreen

### PR DESCRIPTION
**Pull request by Jarru8**
Thanks @bengreenier  😄 

Typescript is not my "First Language", so all good tips to a rookie are welcome 🙇 

Here is my solution to #19 ... window size defaults to primary screens resolution if settings are not found.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -41,18 +41,21 @@ const getTrayIcon = () => {
 const windowDimsSettingsKey = 'overlayed.window'
 
 const allocMainWindow = () => {
+  
+  // create our window
+  mainWindow = new CompositeWindow()
```
Comment (bengreenier) [inline src/index.ts]: __tldr: i'd prefer to put this back to where we pass dimensions directly to the window__

we were actually passing windowDims here for a reason - if we give the correct dimensions (and position) on creation, the window is created at that location. If we refactor to code like this, we'll actually get a flicker where the window is created at the incorrect position (electron default) and then moved to this position with the `setSize` and `setPosition` code below.

File: src/index.ts [hunk 2]
```diff
@@ -41,18 +41,21 @@ const getTrayIcon = () => {
...
 
   // see if we have window settings, if so, use them
-  const windowDims = settings.get(windowDimsSettingsKey, {
-    height: 480,
-    width: 680,
-    x: 0,
-    y: 0,
-  }) as {x: number, y: number, width: number, height: number}
+  // if there is no window settings available 
+  // we read the size of primary display and then adjusting the window size to it
```
Comment (bengreenier) [inline src/index.ts]: nitpick comment: grammar should be "and then adjust the window size to match it"

**Comment by bengreenier** [comment]
@Jarru8 another quick note: i noticed you've got a merge commit in here - are you familiar with [git rebase](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase)? if so it would be great to use that pattern here, as i'd prefer to avoid having a [merge commit](https://stackoverflow.com/questions/47089913/merge-commit-vs-normal-commit) in our history.

let me know if that's a new pattern, happy to work together here!

File: src/index.ts (hunk 2 above)
Comment (Jarru8) [reply src/index.ts]: English is not my first language 😃 feel free to nitpick about my grammar 😄 it only makes it better

File: src/index.ts (hunk 1 above)
Comment (Jarru8) [reply src/index.ts]: Ok, i'll fix these next monday 😄

**Comment by Jarru8** [comment]
> iar w

i'll go back in time, and do the rebase instead 😄

**Pull request closed without merging**